lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -5,7 +5,7 @@
 <%namespace name='static' file='/static_content.html'/>
 <%inherit file="/main.html" />
 
-<%block name="bodyclass">view-teams is-in-course course js</%block>
+<%block name="bodyclass">view-teams is-view-in-course course js</%block>
```

Comment:
Hmm, I'd prefer to use the ``view-*`` prefix for both the single pages as well as for any view-level contexts, so if you're okay with two view classes per page (e.g. ``view-in-course .view-notes``).

The other option is to keep the ``view-*`` classes solely for views/pages, add the ``.is-in-course`` class, and have your Sass/CSS selector be a combination of two classes for the cases you need (``.view-*.is-in-course``).

What do you think? Bad ideas and roll with what you have?